### DISCUSSION TITLE
Need help in Okta-azure password hook

### DISCUSSION THREAD

**Discussion by anugithub16** [community/community#20842] [Other Feature Feedback, Questions, & Ideas]
I need support from scratch for password hook so how could i contact you please send me your email address. Thank you

**Comment by ghost** [comment]
@anugithub16  this will help you 
https://developer.okta.com/blog/2020/09/18/password-hook-migration
https://help.okta.com/oie/en-us/Content/Topics/Directory/password-sync-application.htm
https://developer.okta.com/docs/reference/password-hook/#removing-password-from-existing-user-store
